src/modules/rangeproof/rangeproof.h

```diff
@@ -12,6 +12,79 @@
 #include "ecmult.h"
 #include "ecmult_gen.h"
 
+/** Structure representing data directly encoded into a rangeproof header
+ *
+ * A rangeproof is a proof, associated with a Pedersen commitment, that a
+ * "proven value" in is the range [0, 2^mantissa]. The committed value is
```

Comment:
Shouldn't it be `range [0, 2^mantissa-1]`? If so, then same for `max_value` doc below.